evaluate/report/csv.go

```diff
@@ -59,3 +59,17 @@ func (a AssessmentPerModelPerLanguagePerRepository) Rows() (rows [][]string) {
 
 	return rows
 }
+
+// Header returns the header description as a CSV row.
+func (a AssessmentPerModel) Header() (header []string) {
+	return []string{"model", "score"}
+}
+
+// Rows returns all data as CSV rows.
+func (a AssessmentPerModel) Rows() (rows [][]string) {
+	for model, assessment := range a {
+		rows = append(rows, []string{model.ID(), strconv.FormatUint(uint64(assessment.Score()), 10)})
```

Comment:
I think it would be nice to still have all the metrics and not just the score. Just to have as much data as possible.

Reply:
I've based it upon what we do have now in the csvs, but sure can do that.